wannierberri/symmetry.py

```diff
@@ -110,7 +110,9 @@ def transform_tensor(self, data, rank, TRodd=False, Iodd=False, TRtrans=False):
             res = self.rotate(
                 res.transpose(tuple(range(i)) + tuple(range(i + 1, dim))
                               + (i, ))).transpose(tuple(range(i)) + (dim - 1, ) + tuple(range(i, dim - 1)))
-        if (self.TR and TRodd) != (self.Inv and Iodd):
+        if self.TR and TRodd:
+            res = -res.conj()
+        if self.Inv and Iodd:
             res = -res
```

Comment:
> I updated the code to apply complex conjugation regardless of `TRodd`.

Doesn't it break the optical  conductivity? As I understand, it has symmetric (S) and antisymmetric (A)(in a<->b) components, and both of them are complex.   Under TRS, S is even while A is odd. so we add a transpose... 

Here smth different happens ?

Reply:
In that case, applying complex conjugation once more if `self.TR and TRtrans` would work.

Reply:
Ok, I did that and it fixed the optical conductivity. Now what about SHC?

Complex conjugation of the equation (1) in JiHoon's paper
![image](https://user-images.githubusercontent.com/22172977/206760003-2a7e0167-bb44-4adf-8369-64a7e3757922.png)
acts only on the infinitesimal $i\delta$, so it is equivalent to $\omega \rightarrow -\omega$. 

This kind of makes sense because TR flips frequency, but we do not typically evaluate negative frequencies. 

@jaemolihm , @manxkim  any ideas on that?